From foo@baz Wed Mar 11 14:17:37 PDT 2009
Date: Wed, 11 Mar 2009 14:17:37 -0700
To: Greg KH <greg@kroah.com>
From: Bernd Porr <BerndPorr@f2s.com>
Subject: Staging: comedi: usbduxfast: fix run-time error

If debugging is enabled, this printk will oops, fix that issue.


From: Bernd Porr <BerndPorr@f2s.com>
Cc: Ian Abbott <abbotti@mev.co.uk>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: David Schleef <ds@schleef.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/usbduxfast.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/staging/comedi/drivers/usbduxfast.c
+++ b/drivers/staging/comedi/drivers/usbduxfast.c
@@ -498,8 +498,7 @@ static int usbduxfastsub_upload(struct u
 	int ret;
 
 #ifdef CONFIG_COMEDI_DEBUG
-	printk(KERN_DEBUG "comedi%d: usbduxfast: uploading %d bytes",
-		udfs->comedidev->minor, len);
+	printk(KERN_DEBUG "comedi: usbduxfast: uploading %d bytes", len);
 	printk(KERN_DEBUG " to addr %d, first byte=%d.\n",
 		startAddr, local_transfer_buffer[0]);
 #endif
